src/core/QueryManager.ts

```diff
@@ -165,8 +165,8 @@ export class QueryManager<TStore> {
 
   public async mutate<
     TData,
-    TVariables,
-    TContext,
+    TVariables extends OperationVariables,
+    TContext extends {},
```

Comment:
```suggestion
    TContext extends Record<string, any>,
```

See comment above on the reasoning.